modules/understanding-upgrade-channels.adoc

```diff
@@ -33,15 +60,25 @@ for more build information.
 [id="fast-version-channel_{context}"]
 == fast-{product-version} channel
 
-The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability release. As such, these releases are fully supported, are production quality, and have performed well while available as a release candidate in the `candidate-{product-version}` channel from where they were promoted. Some time after a release appears in the `fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
+The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability (GA) release. The `fast-{product-version}` channel contains releases as soon as their errata are published.
```

Comment:
These two sentences seem to be saying the same thing.  Can we condense to explain that the published errata is how we declare GA?  Something like:

> The `fast-{product-version}` channel is updated with new and previous minor versions of {product-version} as soon as Red Hat declares the given version as a general availability (GA) release by publishing an errata.